packages/select/src/FzSelect.vue

```diff
@@ -66,14 +67,27 @@
       ref="containerRef"
       test-id="fzselect-options-container"
     >
-      <FzSelectOption
-        v-for="option in visibleOptions"
-        :key="option.value"
-        @click="() => handleSelect(option.value)"
-        :option="option"
-        :size="size"
-        :selectedValue="model"
-      />
+      <template v-if="visibleOptions.length">
```

Comment:
instead of doing a `v-if` here, I would assign the default option inside `updateVisibleOption` when no options are available

Reply:
I really don't like to mess with the state of the component... we are not assigning a "real" value here, it's just a visible feedback for the user

Reply:
In any case, the model will have `‘’ ` as its value when there are no options. In the end you have created a ‘button’ with the value `‘’` anyway, so I think it is also more readable to have a single v-for on all options and in the function that populates these options handle the correct case